### PR TITLE
Modify rule S6470(Docker): Improve message

### PR DESCRIPTION
**Pull request by loris-s-sonarsource**
https://sonarsource.atlassian.net/browse/SONARIAC-516

### CODE REVIEW

**Review by egon-okerman-sonarsource — APPROVED** [review]
LGTM, just remove "overly-wide" from both messages.

**Review by loris-s-sonarsource — COMMENTED** [review]
pair review

File: rules/S6470/implementation.adoc [hunk 1]
```diff
@@ -8,11 +8,11 @@ ifdef::env-github,rspecator-view[]
 
 When a dangerous wildcard is found:
 
-* Make sure that using globbing in a (COPY|ADD) source is safe here.
+* Copying using a glob pattern might inadvertently add sensitive data to the container. Make sure it is safe here.
 
 In any other case:
 
-* Make sure that recursively copying directories is safe here.
+* Copying recursively might inadvertently add sensitive data to the container. Make sure it is safe here.
```
Comment (loris-s-sonarsource) [inline rules/S6470/implementation.adoc:15]:
```suggestion
* COPY: Copying recursively might inadvertently add sensitive data to the container. Make sure it is safe here.
* ADD: Adding files recursively might inadvertently add sensitive data to the container. Make sure it is safe here.
```

File: rules/S6470/implementation.adoc [hunk 2]
```diff
@@ -8,11 +8,12 @@ ifdef::env-github,rspecator-view[]
 
 When a dangerous wildcard is found:
 
-* Make sure that using globbing in a (COPY|ADD) source is safe here.
+* Copying using a glob pattern might inadvertently add sensitive data to the container. Make sure it is safe here.
```
Comment (loris-s-sonarsource) [inline rules/S6470/implementation.adoc:11]:
```suggestion
* COPY: Copying using a glob pattern might inadvertently add sensitive data to the container. Make sure it is safe here.
* ADD: Adding files using a glob pattern might inadvertently add sensitive data to the container. Make sure it is safe here.
```

**Pull request merged**